### ISSUE TITLE
FP: Classnamecasing when there is no package

### ISSUE THREAD

**Issue by cverbiest** [Riverside-Software/sonar-openedge#521]
Package name  doesn't match directory name error when there is no package.
Both sample are using the same sonar config
source is located in src, src is the source path
```
class UnusedInFalsePositive: // Package name  doesn't match directory name src/UnusedInFalsePositive.cls    UnusedInFalsePositive.cls
/* */
end class.
```

source is located in src/be, src is the source path
```
class be.UnusedInFalsePositive: // No error shown
/* */
end class.
```

**Comment by gquerret** [comment]
Is that in SonarLint or in the standard analyzer ?

**Comment by cverbiest** [comment]
@gquerret Detected in PDSOE Sonarlint, not tested in standard analyzer

**Comment by gquerret** [comment]
Reproduced in both environment. However, this code should trigger 'No package name'...
